src/components/summary-list/index.md.njk

```diff
@@ -30,24 +30,31 @@ Do not use it for tabular data or a simple list of information or tasks, like a
 
 There are 2 ways to use the summary list component. You can use HTML or, if you’re using [Nunjucks](https://mozilla.github.io/nunjucks/) or the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com), you can use the Nunjucks macro.
 
-### Summary list with actions
+{{ example({group: "components", item: "summary-list", example: "without-actions", html: true, nunjucks: true, open: false}) }}
+
+###  Adding actions to each row
 
 You can add actions to a summary list, like a ‘Change’ link to let users go back and edit their answer.
 
 For sighted users, the actions get their context from the other content in the row they appear in.
 
 Assistive technology users, like those who use a screen reader, may hear the links out of context and not know what they do.
+
 To give more context, add visually hidden text to the links. This means a screen reader user will hear a meaningful action, like ‘Change name’ or ‘Change date of birth’.
 
 {{ example({group: "components", item: "summary-list", example: "default", html: true, nunjucks: true, open: false, titleSuffix: "second"}) }}
 
-### Summary list without actions
+### Adding actions to only some rows
```

Comment:
Alternatives?
- Adding actions to some rows, but not others

Reply:
Is line 49 about adding actions to some rows, or is it more about indicating which rows do not have actions?

Reply:
It's... both!

The way Ollie explained it, if there is no action (link text) for a particularly row, there also needs to be some other changes to make the visually hidden text work correctly (and indicate this to the user).

Reply:
We could potentially leave this heading out entirely – it does follow naturally from the previous section. But makes it hard to deep link to it, for example.

Reply:
Shall we remove the heading @EoinShaughnessy? 
I do agree it follows naturally and the heading might be cause more confusion than it's worth. 

Also think the overall H3 seems to still work as-is.

Reply:
Yeah, reckon we should nix the heading. 👍🏻